nixos/modules/services/torrent/opentracker.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.opentracker;
+
+  out = {
+    options.services.opentracker = api;
+    config = lib.mkIf cfg.enable imp;
+  };
+
+  api = {
+    enable = mkEnableOption "opentracker";
+
+    package = mkOption {
```

Comment:
But why would you want that in this case. That's more important than what other modules are doing.